tools/build.py

```diff
@@ -82,6 +81,10 @@ def init_options():
     parser.add_argument('--clean', action='store_true', default=False,
         help='Clean build directory before build (default: %(default)s)')
 
+    parser.add_argument('--config', default=path.BUILD_CONFIG_PATH,
+        help='Specify the config file (default: %(default)s)',
+        dest='config_path')
+
```

Comment:
It seems almost all arguments in ``tools/build.py`` are matched the members in ``build.config``. It would be consistent to add "config" in "build_option" in ``build.config`. Please put the item in alphabetical order although some items break the rules.